masterlist.yaml

```diff
@@ -34013,3 +34020,16 @@ plugins:
         util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 4
         udr: 40
+
+  - name: 'ImpeREAL Empire Unique Cities - Sutch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/37867/'
+        name: 'ImpeREAL Empire Unique Cities - Sutch'
+    dirty:
+      - <<: *quickClean
+        crc: 0xEB59AB31
+        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 2
+    clean:
+      - crc: 0x0872D0D1
+        util: 'TES4Edit v4.0.4'
```

Comment:
```suggestion
```

Explained in #1621.

Reply:
This will come out after rebasing e7d7d9b917d82bbdda0448d8c3af81a9e48f3f67 on the final commit for #1621